src/ILLink.Shared/ILLink.xsd

```diff
@@ -0,0 +1,193 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would rename the file to `ILLink.Descriptor.xsd`

Reply:
I made this schema broad enough to cover `ILLink.Substitutions.xml` as well as `ILLink.Descriptor.xml` and `ILLink.LinkAttributes.xml`, so I don't want to name it just for descriptors, but if we don't like `ILLink.xsd`, maybe something like `ILLink.General.xsd`?

Reply:
This one I started as a form of documentation before I realized how different each xml file is. I think that's a good point, this schema is a little too general to provide any value. I'll remove this file and only do the more specific schema.